docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+
+services:
+  app:
+    build: .
+    image: electic-io
+    container_name: electric-io
+    command: npm run start
+    ports:
+      - 3000:3000
+    env_file:
+      - .env
+    environment:
+      - SIMULATING
+    volumes: # For file editing, watching, etc
```

Comment:
This preserves the changes made in the container back to your host. And also changes (as you're developing) on your host get reflected back into the running app container.